lib/types/marks.dart

```diff
@@ -57,6 +57,7 @@ class MARKS {
       fontWeight: textStyles['bold']?.fontWeight,
       fontStyle: textStyles['italic']?.fontStyle,
       decoration: textStyles['underline']?.decoration,
+      fontFamily: textStyles['bold']?.fontFamily ?? textStyles['italic']?.fontFamily,
```

Comment:
Something like the following may be able to replace the `getMarksTextStyles` method completely and maintain styles set on a custom RenderMark while still explicitly setting the 3 specific styles (bold, italic, underline) - I haven't tested it yet though. Would you be willing to give it a try for your use case and see if it'll maintain your fontFamily TextStyle settings for bold and italic if they are passed via a custom RenderMark?

```
static TextStyle getMarksTextStyles(List<Mark> marks, Map<dynamic, TextStyle> renderMark) {
  TextStyle combinedStyle = TextStyle();
  TextStyle combinedTypeStyle = TextStyle();

  marks.forEach((Mark mark) {
    if (renderMark.containsKey(mark.type)) {
      TextStyle markStyle = renderMark[mark.type]!;
      combinedStyle = combinedStyle.merge(markStyle);
      combinedTypeStyle = combinedTypeStyle.copyWith(
        fontWeight: mark.type == 'bold' ? markStyle.fontWeight : combinedTypeStyle.fontWeight,
        fontStyle: mark.type == 'italic' ? markStyle.fontStyle : combinedTypeStyle.fontStyle,
        decoration: mark.type == 'underline' ? markStyle.decoration : combinedTypeStyle.decoration,
      );
    }
  });

  return combinedTypeStyle.merge(combinedStyle);
}
```

Reply:
Yes, it works great! sorry for late reply:)